samples/WOCCatalog/WOCCatalog/DisplayModeViewController.m

```diff
@@ -307,7 +307,7 @@ - (id)init {
     cell.detailTextLabel.text = @"0.0 = fit to window";
     [self.rows addObject:cell];
 
-    fixedAspect = [[UITextField alloc] initWithFrame:CGRectMake(0, 0, 100, 20)];
+    fixedAspect = [[UITextField alloc] initWithFrame:CGRectMake(0, 0, 100, 25)];
```

Comment:
>25 [](start = 75, length = 2)

stealthing in a fix for poorly-rendered UITextFields in this view of the app, as 20 is not high enough to properly-render a Xaml textbox (it works, but it looks terrible and is vertically truncated).